UnitTests/Views/ColorPickerTests.cs

```diff
@@ -1,79 +1,652 @@
-﻿namespace Terminal.Gui.ViewsTests;
+﻿using Xunit.Abstractions;
+using Color = Terminal.Gui.Color;
+
+namespace UnitTests.Views;
 
 public class ColorPickerTests
 {
     [Fact]
-    public void Constructors ()
+    [AutoInitShutdown]
```

Comment:
I've worked really hard to make it so it's VERY rare where `AutoInitShutdown` is needed for a test. 

Please don't use it unless absolutely needed.  When used, if a test fails, it makes it very hard to find the failure because app state is not reset.

Here's how to fix this test as an example:

```cs
    [Fact]
    [SetupFakeDriver]
    public void ColorPicker_Construct_DefaultValue ()
    {
        var cp = new ColorPicker { Width = 20, SelectedColor = new (0, 0) };

        cp.Style.ShowTextFields = false;
        cp.ApplyStyleChanges ();

        // Should be only a single text field (Hex) because ShowTextFields is false
        Assert.Single (cp.Subviews.OfType<TextField> ());

        cp.Draw ();

        // All bars should be at 0 with the triangle at 0 (+2 because of "H:", "S:" etc)
        var h = GetColorBar (cp, ColorPickerPart.Bar1);
        Assert.Equal ("H:", h.Text);
        Assert.Equal (2, h.TrianglePosition);
        Assert.IsType<HueBar> (h);

        var s = GetColorBar (cp, ColorPickerPart.Bar2);
        Assert.Equal ("S:", s.Text);
        Assert.Equal (2, s.TrianglePosition);
        Assert.IsType<SaturationBar> (s);

        var v = GetColorBar (cp, ColorPickerPart.Bar3);
        Assert.Equal ("V:", v.Text);
        Assert.Equal (2, v.TrianglePosition);
        Assert.IsType<ValueBar> (v);

        var hex = GetTextField (cp, ColorPickerPart.Hex);
        Assert.Equal ("#000000", hex.Text);
    }
```